### PR TITLE
NMS-16133: Add language to docs for how to find schema to Kafka Producer

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-16133

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/kafka-producer/kafka-producer.adoc [hunk 1]
```diff
@@ -66,3 +66,7 @@ The records are keyed in this fashion to help ensure that collection sets relate
 When enabled (this functionality is disabled by default), the metrics are written to a topic named `metrics`.
 
 NOTE: When exclusively writing to Kafka, no metrics or resource graphs will be available on the {page-component-title} instance.
+
+== Schema
+View the https://github.com/OpenNMS/opennms/blob/develop/features/kafka/producer/src/main/proto/opennms-kafka-producer.proto[Kafka Producer schema] on GitHub.
+Change branches or tags to see a version from that point in time.
```
Comment (marshallmassengill) [inline docs/modules/operation/pages/deep-dive/kafka-producer/kafka-producer.adoc:72]: It  might be good to include a list or some information about how to go from release version to branch but this does get the general process across.

**Review by marshallmassengill — APPROVED** [review]
Good to me.  I did add a comment for including a list or some information about how to go from release version to branch but I think this at least has some additional detail.

**Comment by Bonrob2** [comment]
> Good to me. I did add a comment for including a list or some information about how to go from release version to branch but I think this at least has some additional detail.

There's probably a better place to put the info on navigating the branches, but for now I'll include it here, until we can look over the docs and determine a better location.

**Review by marshallmassengill — APPROVED** [review]
LGTM!

**Review by marshallmassengill — APPROVED** [review]
LGTM!

**Review by marshallmassengill — APPROVED** [review]
LGTM (What's the definition of insanity?)

**Review by marshallmassengill — APPROVED** [review]
Once more unto the breach!  Or something.  This looks great!

**Comment by Bonrob2** [comment]
@mmahacek , adding you here, as even though Marshall has approved this several times, GitHub isn't accepting that as sufficient for me to merge.

**Review by mmahacek — APPROVED** [review]

**Pull request merged**